mne/minimum_norm/tests/test_resolution_matrix.py

```diff
@@ -0,0 +1,107 @@
+"""
+Test the following properties of the resolution matrix.
+
+Resolution matrix is symmetrical for MNE (fixed and free source orientation).
+Resolution matrix has zero dipole localisation error for columns (PSFs).
+"""
+
+import os.path as op
+import numpy as np
+from numpy.testing import (assert_array_almost_equal, assert_array_equal)
+
+import mne
+from mne.datasets import testing
+from mne.minimum_norm.resolution_matrix import make_resolution_matrix
+
+data_path = testing.data_path(download=False)
+subjects_dir = op.join(data_path, 'subjects')
+fname_inv = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-meg-eeg-oct-6-meg-inv.fif')
+fname_evoked = op.join(data_path, 'MEG', 'sample',
+                       'sample_audvis_trunc-ave.fif')
+fname_raw = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc_raw.fif')
+fname_t1 = op.join(data_path, 'subjects', 'sample', 'mri', 'T1.mgz')
+fname_src = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-meg-eeg-oct-6-fwd.fif')
+fname_src_fs = op.join(data_path, 'subjects', 'fsaverage', 'bem',
+                       'fsaverage-ico-5-src.fif')
+fname_src_3 = op.join(data_path, 'subjects', 'sample', 'bem',
+                      'sample-oct-4-src.fif')
+fname_stc = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc-meg')
+fname_vol = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-grad-vol-7-fwd-sensmap-vol.w')
+fname_vsrc = op.join(data_path, 'MEG', 'sample',
+                     'sample_audvis_trunc-meg-vol-7-fwd.fif')
+fname_inv_vol = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc-meg-vol-7-meg-inv.fif')
+rng = np.random.RandomState(0)
+
+fname_fwd = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-meg-eeg-oct-4-fwd.fif')
+fname_cov = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc-cov.fif')
+
+# get functions for resolution matrix, metrics etc.
+
+
+@testing.requires_testing_data
+def test_resolution_matrix():
+    """Test whether MNE's resolution matrix is symmetric."""
```

Comment:
you test more than this. Just say
```suggestion
    """Test make_resolution_matrix function."""
```